packages/xstate-test/src/machine.ts

```diff
@@ -103,20 +103,6 @@ export function createTestModel<TMachine extends AnyStateMachine>(
             );
           })
         ),
-      testTransition: async (step) => {
```

Comment:
Removed testTransition from options. According to our discussion it might be config overkill.